catalog/plugin/catalog-plugin-query-monitor-commands/src/main/java/ddf.catalog.querymonitorcommands/RemoveSearchAfterCompleteCommand.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
```

Comment:
The package structure for this module is odd, the folder `ddf.catalog.querymonitorcommands/` should actually be multiple folders `ddf/catalog/querymonitorcommands`